kcidb/misc.py

```diff
@@ -187,6 +208,14 @@ def argparse_output_add_args(parser):
              'RFC 7464 and "application/json-seq" media type.',
         action='store_true'
     )
+    parser.add_argument(
+        'schema_version',
+        metavar='Schema Version',
+        type=schema_versions,
+        help='KCIDB schema version. Default is LATEST',
+        nargs='?',
+        default="LATEST",
+    )
```

Comment:
This wouldn't be the right place to add the argument specification, as this function is used to setup command-line parsing for commands outputting **any** JSON, not just the schema.

In general, it's good practice to look for all ways the function is used before making changes like that, to not accidentally introduce unwanted behavior.